src/app/organization/tellers/create-teller/create-teller.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="tellerForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Teller Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="tellerForm.controls.name.hasError('required')">
+              Teller Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Office</mat-label>
+            <mat-select required formControlName="officeId">
+              <mat-option *ngFor="let office of officeData" [value]="office.id">
+                {{ office.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="tellerForm.controls.officeId.hasError('required')">
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Description</mat-label>
+            <input matInput formControlName="description">
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Start Date</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="startDatePicker" required formControlName="startDate">
+            <mat-datepicker-toggle matSuffix [for]="startDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #startDatePicker></mat-datepicker>
+            <mat-error *ngIf="tellerForm.controls.startDate.hasError('required')">
+              Start Date is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>End Date</mat-label>
+            <input matInput [min]="minDate" [matDatepicker]="endDatePicker" formControlName="endDate">
+            <mat-datepicker-toggle matSuffix [for]="endDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #endDatePicker></mat-datepicker>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Status</mat-label>
+            <mat-select required formControlName="status">
+              <mat-option value="300">
+                Active
+              </mat-option>
+              <mat-option value="2">
+                Inactive
```

Comment:
I personally would make this a checkbox, with the value being set to false as default, and during the submission of the form, convert the boolean values in the FormControl into the IDs (2 and 300).

Reply:
but in the community app, the status is in the form as a select option.

Reply:
I am aware of that. However, as it is a boolean value, it would be best for the UI wise to make it a checkbox and name the label 'Is Active'. I believe that things are not completely set in stone for the implementation as long as it provides good user experience. @abhaychawla What are your views on this?